### PR TITLE
Playlist

### PR DESCRIPTION
**Pull request by Tejinder15**
# Done With Playlist Management

### CODE REVIEW

**Review by dipteekhd — APPROVED** [review]
Functionality working fine! Very Nice UI!
You can create service file for all API (Suggestion)

File: src/Reducer/playlist-reducer.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+const initialPlaylistState = { playlist: [] };
+
+const playlistReducer = (state, action) => {
+  switch (action.type) {
+    case "REMOVE_FROM_PLAYLIST":
+      return {
+        playlist: state.playlist.map((item) =>
```
Comment (dipteekhd) [inline src/Reducer/playlist-reducer.js:7]: {...state,playlist : logic} copy state value and then update value

File: src/App.js [hunk 2]
```diff
@@ -14,9 +14,11 @@ function App() {
         <Route path="/signup" element={<Signup/>}/>
         <Route path="/videos" element={<Videos/>}/>
         <Route path="/watch" element={<Watch/>}/>
+        <Route path="*" element={<h1 className="page_title center-text">There's Nothing Here!</h1>}/>
```
Comment (gadgetfather) [inline src/App.js:17]: You can add button on this component , to take user back to the home page

**Review by gadgetfather — APPROVED** [review]
Functionality is working fine. when the specific playlist is opened add that playlist name on top so the user will know which playlist he/she is in.

**Pull request merged**